### ISSUE TITLE
Add Speed [ SPED ] axis

### ISSUE THREAD

**Issue by vv-monsalve** [googlefonts/axisregistry#136]
### Requirements

- [x] I have inspected the current [Axis Registry](https://github.com/googlefonts/axisregistry/tree/main/Lib/axisregistry/data) and there is not a registered custom axis that could be used for the variation purpose on the upcoming font project.
- [x] The metadata fields of the proposed axis meet the *Axis Requirements* as specified in the Protocol (linked above).

### Font project(s) using the axis

Playpen, currently in a private repo

### Short description of what the axis does

As the speed increase, the joins and ending strokes go from slow (0%), with more rounded arches, to fast (100%), making the junctions more diagonal and fluid.

### Image

![PlaypenSans](https://github.com/googlefonts/axisregistry/assets/48698976/c5f77d33-c37c-454d-a1a4-534b9b2bf2a5)

### Why is the axis needed

Speed is a decisive factor in the construction of the letters, which alters their shape and, therefore, the rhythm and fluidity of the writing. This axis would offer specific control over that factor, allowing a fine-tuning of those features.

### Axis metadata fields

```
# SPED based on github.com/googlefonts/playpen
tag: "SPED"
display_name: "Speed"
min_value: 0
default_value: 0
max_value: 100
precision: 
fallback {
  name: "Default"
  value: 0.00
}
fallback_only: false
description: "As the speed increase, the joins and ending strokes go from slow (0%),"
  " with more rounded arches to fast (100%), making the junctions more diagonal and fluid."
````

**Comment by vv-monsalve** [comment]
@josescaglione Please feel free to make adjustments.

**Comment by josescaglione** [comment]
maybe we can improve description:

The velocity at which writing is executed significantly impacts the tension of curves and the angularity of connections. When writing at a low speed (0%), arches tend to exhibit roundness, and curve radii remain more open. Conversely, a high speed (100%) results in sharper turns, acute connections, and the manifestation of diagonal and fluid strokes.

**Comment by davelab6** [comment]
We want this description to be rather short, as its used in a tooltip on the fonts.google.com "type tester", while a much longer description is desirable for the GFK Glossary page on the axis.

**Comment by josescaglione** [comment]
The velocity at which writing is executed significantly impacts the tension of curves and the angularity of connections. 

??

**Comment by vv-monsalve** [comment]
As the Speed of writing increases, it impacts the curves and angularity of connections. Going from low-speed roundness arches (0%) to sharper turns with acute connections at high speed (100%), manifesting more fluidity.

**Comment by davelab6** [comment]
I would rephrase as:

> Adjusts the apparent velocity of writing the letterforms, by increasing curve tension and the angularity of stroke connections, from rounder arches to sharper turns.

Here's a couple of in-context examples:

<img width="593" alt="Screenshot 2023-06-22 at 12 35 51" src="https://github.com/googlefonts/axisregistry/assets/261579/c45fcb14-0f85-4cc9-b7d9-733889ac9b47">

<img width="547" alt="Screenshot 2023-06-22 at 12 36 22" src="https://github.com/googlefonts/axisregistry/assets/261579/f33cdbc4-8b07-451d-98ba-1bdf55d19a1f">

**Comment by vv-monsalve** [comment]
Final Metadata fields:

```
# SPED based on github.com/googlefonts/playpen
tag: "SPED"
display_name: "Speed"
min_value: 0
default_value: 0
max_value: 100
precision: 
fallback {
  name: "Default"
  value: 0.00
}
fallback_only: false
description: "Adjusts the apparent speed of writing the letterforms by increasing curve tension and the angularity"
    " of stroke connections, from low-speed round arches (0%) to sharper turns at high-speed (100%)."
````

**Comment by vv-monsalve** [comment]
We'll add it in the future when the big PW family is ready to be onboarded.

**Comment by davelab6** [comment]
The https://github.com/TypeTogether/Playwrite 'parent' VF is not useful due to limitations in OpenType Features stylistic set. We'll need to update OpenType format itself to fix this.
